docs/source/api_reference/forecasting.rst

```diff
@@ -335,6 +335,14 @@ Structural time series models
 
     Prophet
 
+.. currentmodule:: sktime.forecasting.prophetverse
+
+.. autosummary::
+    :toctree: auto_generated/
+    :template: class.rst
+
+    ProphetverseUnivariate
```

Comment:
Hi Franz! Just a warning that the old name is still being used inside docs.

Reply:
ah, thx